source/reference/mongosync.txt

```diff
@@ -255,6 +255,12 @@ the destination cluster, ``mongosync`` sets the write concern level to
 For more information on read and write concern configuration and
 behavior, see :ref:`read-concern` and :ref:`write-concern`. 
 
+Read Preference
+~~~~~~~~~~~~~~~
+By default, ``mongosync`` sets the source cluster read preference to
+:readmode:`nearest` to distribute reads evenly across nodes. To change
+the read preference of clusters, specify the read preference mode 
```

Comment:
**Suggestion**

You might want to add some content here on how the user can set the read preference mode.
Also you may want to be explicit and mention "source cluster" similar to `source/includes/read-preference-connection-string.rst`.

Something like:

To change the read preference of the source cluster, specify the read preference mode in the URI of the connection string. For details, see :ref:`connections-read-preference`.

```suggestion
the read preference of clusters, specify the read preference mode.
```